src/application/services/useNote.ts

```diff
@@ -270,6 +277,22 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     parentNote.value = undefined;
   }
 
+  /**
+   * Set parent for the note
+   * @param id - Child note id
+   * @param newParentURL - New parent note URL
+   */
+  const setParent = async (id: NoteId, newParentURL: string): Promise<void> => {
```

Comment:
use named function instead of anonymous, it's better for debugging
```suggestion
  async function setParent(id: NoteId, newParentURL: string): Promise<void> {
```

Reply:
I'd suggest leaving `setParent` at the `useNoteSettigns` service because we can change parent only from the settings page